be/src/exprs/vectorized/runtime_filter.h

```diff
@@ -423,6 +413,16 @@ class RuntimeBloomFilter final : public JoinRuntimeFilter {
         }
     }
 
+    void evaluate_min_max(const CppType* values, uint8_t* selection, size_t size) const {
+        if constexpr (!IsSlice<CppType>) {
+            for (size_t i = 0; i < size; i++) {
+                selection[i] = (values[i] >= _min && values[i] <= _max);
```

Comment:
```
selection[i] = (values[i] >= _min & values[i] <= _max);
```

Reply:
I'v checked with assmebly code with following function, they are the same.

```
void ff(char* res, int* data, int size, int _min_value, int _max_value) {
            for (int i = 0; i < size; i++) {
 //                 res[i] = (data[i] >= _min_value && data[i] <= _max_value);
                res[i] = (data[i] >= _min_value & data[i] <= _max_value);
            }
}
```